source/includes/fact-navigate-to-settings.rst

```diff
@@ -0,0 +1,15 @@
+To open the |compass| Settings Panel, click the gear icon on the left 
+column of the |compass-short| home screen.
+
+.. figure:: /images/compass/settings-gear-icon.png
+   :alt: Settings gear icon location on MongoDB Compass UI
```

Comment:
small nit, but do you think we can resize this image to be smaller? I don't think it needs to take up as much visual real estate as it currently does. You can adjust this with the `:width:` flag